firefox-ios/Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -74,9 +73,7 @@ class ZoomPageBar: UIView, ThemeApplicable, AlphaDimmable {
     }
 
     private lazy var zoomLevel: UILabel = .build { label in
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .callout,
-                                                            size: UX.fontSize,
-                                                            weight: .semibold)
+        label.font = FXFontStyles.Regular.body.scaledFont()
```

Comment:
It looks like we may want the bold styles since the weight is semibold and use callout type here.

`FXFontStyles.Bold.callout.scaledFont()`

Could you provide updated screenshots for @cwzilla to validate? Thank you, appreciate you working with me on these iterations!

Reply:
Reviewed with @cwzilla, designs look good with updates.